apps/web/src/shared/components/layout/NavBadge.tsx

```
@@ -0,0 +1,38 @@
+    "absolute -top-1 -right-1 z-10 flex min-w-[16px] h-4 items-center justify-center rounded-full px-1 text-[9px] font-bold leading-none tabular-nums pointer-events-none";
+
+  const colors =
+    variant === "red"
+      ? "bg-destructive text-white"
+      : "bg-muted text-muted-foreground border border-border/60";
+
+  return (
+    <span
+      className={`${base} ${colors} ${className}`}
+      aria-label={`${count} notification${count !== 1 ? "s" : ""}`}
+    >
```

Biome (2.4.4)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)